File: core/src/main/java/com/uiptv/service/AccountPlayerService.java
package com.uiptv.service;

import com.uiptv.model.Account;
import com.uiptv.model.Channel;
import com.uiptv.model.PlayerResponse;

import java.io.IOException;

public interface AccountPlayerService {
    PlayerResponse get(Account account, Channel channel, String series, String parentSeriesId, String categoryId) throws IOException;
}

File: server/src/main/kotlin/com/uiptv/service/AccountPlayerService.kt
package com.uiptv.service

import com.uiptv.model.Account
import com.uiptv.model.Channel
import com.uiptv.model.PlayerResponse
import java.io.IOException

interface AccountPlayerService {
    @Throws(IOException::class)
    fun get(account: Account, channel: Channel, series: String?, parentSeriesId: String?, categoryId: String?): PlayerResponse
}
